docs/core-concepts/app-types.md

```diff
@@ -0,0 +1,125 @@
+# Types of portability in .NET Core
+
+## Introduction
+There are several ways to think about the "types" of applications you can build. Usually, the types describe 
+a certain execution model or are based on what the application can "do"; examples of these are "console application", 
+"web application" etc. All of these types of applications (and more) can be created with .NET Core, since it is 
+a general purpose development platform.
+
+However, given its unique and cross-platform nature, .NET Core also has another angle through which to observe the type of the application 
+and that is application's *portability*. Portability here means, essentially, where can you run your application and what are the 
```

Comment:
@mairaw ah, ok. I guess I was dizzy when reading the feedback, I read it as something else. :)